src/EnergyPlus/InternalHeatGains.cc

```diff
@@ -983,14 +1002,33 @@ namespace InternalHeatGains {
                                         }
                                     }
                                 }
-                            } else if (MustInpSch && (People(Loop).Fanger || People(Loop).Pierce || People(Loop).KSU)) {
+                            } else if (MustInpSch && ThermalComfortRuns) {
                                 if (Item1 == 1) {
                                     ShowSevereError(state, RoutineName + CurrentModuleObject + "=\"" + AlphaName(1) + "\", blank " + cAlphaFieldNames(13) +
                                                     " is required for this item.");
                                     ErrorsFound = true;
                                 }
                             }
 
+                            // TODO: why Item1 == 1 (first People object) required in all People input error logging?
```

Comment:
Yes, this is confusing, but it's ok (it took me a few minutes to see this). Item1 == 1 refers to the first Zone for this People object. 
`for (Item1 = 1; Item1 <= PeopleObjects(Item).NumOfZones; ++Item1) {`
If a given People object references a ZoneList, the fields in the object get processed multiple times, so it makes sense to throw warnings/errors only on the first pass.
I could think of better variable names that `Item1` here, but probably not worth changing at this point.